assembly/src/parsers/u32_ops.rs

```diff
@@ -568,21 +559,19 @@ pub fn parse_u32lte(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Ass
 /// Specifically we test the first two numbers to be u32, then perform a lt check with the
 /// numbers swapped.
 ///
-/// TODO: add VM cycle count.
+/// VM cycles per mode:
+/// u32gt: 12 cycles
+/// u32gt.unsafe: 6 cycles
 pub fn parse_u32gt(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), AssemblyError> {
-    match op.num_parts() {
-        0 => return Err(AssemblyError::missing_param(op)),
-        1 => {
-            // make sure the first two values on the stack are u32 (while sapping their order)
-            assert_u32_operands(span_ops, false);
-
-            // We skip the swap which reverses the order of the numbers,
-            // so a lt check here becomes gt.
-            compute_lt(span_ops);
-        }
-        _ => return Err(AssemblyError::extra_param(op)),
+    handle_u32_and_unsafe_check(span_ops, op, false)?;
+    if op.num_parts() == 2 {
+        // Swap the order in unsafe mode since we only swap on u32 check.
+        span_ops.push(Operation::Swap);
     }
```

Comment:
Couldn't we just pass *true* as the last parameter to `handle_u32_and_unsafe_check()` and avoid this *if* statement?